openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/WbPanel.html

```diff
@@ -195,12 +195,12 @@
 		<div id="wb-formula" class="wb-formula ui-corner-all ui-widget-content">
 			<div wicket:message="title:wb.tool.math.formula" class="header ui-dialog-titlebar ui-corner-all ui-widget-header ui-helper-clearfix ui-draggable-handle">
 				<span class="ui-dialog-title"><wicket:message key="wb.tool.math.formula"/></span>
-				<button type="button" class="ui-button ui-corner-all ui-widget ui-button-icon-only ui-dialog-titlebar-close" wicket:message="title:85">
-					<span class="ui-button-icon ui-icon ui-icon-closethick"></span>
-					<span class="ui-button-icon-space"> </span>
-					<wicket:message key="85"/>
-				</button>
 			</div>
+			<button type="button" class="ui-button ui-corner-all ui-widget ui-button-icon-only ui-dialog-titlebar-close" wicket:message="title:85">
```

Comment:
I don't quite understand why we manually code this button instead of $jquery.dialog() but its fine for me for now. Probably because of the custom resizeable I guess.

Reply:
I would change this button to be bootstrap button, but this is cosmetics

Reply:
Is there an example somewhere ?

Reply:
This might be an example:
https://github.com/apache/openmeetings/blob/master/openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/WbPanel.html#L49

Reply:
ok thanks, I'll have a look. But prob in another PR.